src/prefect/runner/runner.py

```diff
@@ -550,13 +552,23 @@ async def stop(self):
             )
 
     async def execute_flow_run(
-        self, flow_run_id: UUID, entrypoint: Optional[str] = None
-    ) -> None:
+        self,
+        flow_run_id: UUID,
+        entrypoint: str | None = None,
+        command: str | None = None,
+        cwd: Path | None = None,
+        env: dict[str, str | None] | None = None,
+        task_status: anyio.abc.TaskStatus[int] | None = None,
+        stream_output: bool = True,
+    ) -> anyio.abc.Process | None:
```

Comment:
I expanded this interface to include the aspects that can be customized for a process work pool.